tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Order Update Party7171.feature

```diff
@@ -0,0 +1,85 @@
+﻿Feature: Updating the Ordering Party in a Buyer Section

```

Comment:
As per comments from the code standards meeting on Friday, the number in the filename can be removed